README.md

```diff
@@ -586,6 +586,111 @@ crud.len('customers')
 ...
 ```
 
+### Statistics
+
+`crud` routers provide statistics on called operations.
+```lua
+-- Returns table with statistics information.
+crud.stats()
+```
+
+Statistics collect enabled by default on all routers.
+It can be disabled and re-enabled later.
+```lua
+-- Disables statistics collect and resets all collectors.
+crud.disable_stats()
+
+-- Enable statistics collect and recreates all collectors.
+crud.enable_stats()
+```
+
+If [`metrics`](https://github.com/tarantool/metrics) found,
+you can use metrics collectors to store statistics
+instead of local collectors.
+It is required to use version `0.9.0` or greater,
+otherwise local collectors will be used.
+```
+-- Use metrics collectors.
+crud.enable_stats({ driver = 'metrics' })
+```
+By default, local collectors (`{ driver = 'local' }`)
+are used. Metrics collectors are much sophisticated and
+would show execution time quantile with aging for calls.
+Be wary that computing quantiles may affect overall
+performance under high load. Using local
+collectors or disabling stats is an option if
+you want to prioritize performance.
+
+Enabling stats on non-router instances is meaningless.
+
+`crud.stats()` contains several sections: `insert` (for `insert` and `insert_object` calls),
+`get`, `replace`  (for `replace` and `replace_object` calls), `update`,
+`upsert` (for `upsert` and `upsert_object` calls), `delete`,
+`select` (for `select` and `pairs` calls), `truncate`, `len` and
+`borders` (for `min` and `max` calls).
+
+```lua
+crud.stats()['insert']
+---
+- ok:
+    latency: 0.002
+    count: 19800
+    time: 39.6
+  error:
+    latency: 0.000001
+    count: 4
+    time: 0.000004
+---
+```
+Each section contains different collectors for success calls
+and error (both error throw and `nil, err`) returns. `count`
+is total requests count since instance start or stats restart.
+`latency` is 0.99 quantile of request execution time if `metrics`
+driver used, otherwise `latency` is total average.
+`time` is total time of requests execution.
+
+In `metrics` registry statistics are stored as `tnt_crud_stats` metrics
+with `operation` and `status` label_pairs.
+```
+metrics:collect()
+---
+- - label_pairs:
+      status: ok
+      operation: insert
+    value: 221411
+    metric_name: tnt_crud_stats_count
```

Comment:
There is no information about space name. So we can call such stats as "среднее по больнице".

Reply:
Well, tarantool `box.stat()` also aggregates all operation metrics. Since everything is in out hands, we can make it, but it will make structure is more complex (`crud.stats()`) and will create more time series in Prometheus/InfluxDB in case of an export. I'm not voting against it, but do we really need this info?

Reply:
> Well, tarantool box.stat() also aggregates all operation metrics.

This is a fatal flaw that does not make users' lives easier. You can ask other persons that interested in this feature and I guess they want per space stats.

Reply:
Ok, that's fair

Reply:
Reworked stats structure to contain info about space name.